server/server.go

```diff
@@ -76,18 +76,39 @@ type Command struct {
 
 	Handler pilosa.Handler
 	ln      net.Listener
+
+	ServerOptions []pilosa.ServerOption
```

Comment:
I see why you exported this, but... we probably shouldn't?  Maybe we could store the options on the test.Main